fastapi/encoders.py

```diff
@@ -14,7 +14,7 @@ def jsonable_encoder(
     include: Union[SetIntStr, DictIntStrAny] = None,
     exclude: Union[SetIntStr, DictIntStrAny] = set(),
     by_alias: bool = True,
-    skip_defaults: bool = False,
+    exclude_unset: bool = False,
```

Comment:
I renamed `skip_defaults` to `exclude_unset` everywhere, since that is the intended usage in the new pydantic version. I think this will be the best approach, but it would involve a little extra migration pain since it would require skip_defaults to be renamed. Since it has precisely the same functionality though that should be a pretty straightforward find-and-replace.

Eventually, I think it will make sense to also support the new `exclude_defaults` argument, but I figured that was better suited for a separate PR.

If you want to keep the argument and raise a deprecation warning (like what pydantic currently does), or just keep the argument named as is and use the new keyword argument when calling to pydantic, I can change it everywhere, just let me know.

Reply:
Agreed! It makes sense to keep in line with Pydantic.

I added some backward compatibility with warnings :nerd_face: